app/javascript/components/applications/application_form/PaperApplicationForm.js

```diff
@@ -113,7 +113,7 @@ class PaperApplicationForm extends React.Component {
                     <button className='primary radius margin-right save-btn' type='submit' onClick={() => this.saveSubmitType('Save', form)} disabled={loading}>
                       {loading ? 'Saving…' : 'Save'}
                     </button>
-                    <button className='primary radius' type='submit' onClick={() => this.saveSubmitType('SaveAndNew')} disabled={loading}>
+                    <button className='primary radius' type='submit' onClick={() => this.saveSubmitType('SaveAndNew', form)} disabled={loading}>
```

Comment:
I believe this button also needs form passed through as well so I added it, not part of the feature but a fix.